packages/cli/src/request.ts

```diff
@@ -51,12 +55,18 @@ export const listen = async (host: string, params?: RequestParams, handlers?: Re
       es.close();
       console.error('connects to daemon timeout, please run "pipcook daemon restart".');
     }, 5000);
-    const onerror = (e: Event) => {
+    const onerror = (e: ErrorEvent) => {
       if (handshaked === false) {
         es.close();
         clearTimeout(timeoutHandle);
-        console.error('daemon is not started, run "pipcook daemon start"');
         es.removeEventListener('error', onerror);
+
+        // print error
+        if (typeof e?.status === 'number') {
+          console.error('occurrs an daemon error with the following response:\n', e);
+        } else {
+          console.error(`daemon is not started(${e.message}), run "pipcook daemon start".`);
```

Comment:
if `e` is undefined, throw an error here.

Reply:
See https://github.com/EventSource/eventsource/blob/master/lib/eventsource.js#L53, https://github.com/EventSource/eventsource/blob/master/lib/eventsource.js#L143 and https://github.com/EventSource/eventsource/blob/master/lib/eventsource.js#L162, the upstream library `eventsource` guarantee this won't be a undefined value.

Reply:
If `e` is not undefined, why we check it by `if (typeof e?.status === 'number') {`?